### PR TITLE
test: add new test 'Test Browser Context Viewport'

### PR DESCRIPTION
**Pull request by ephung01**

### CODE REVIEW

File: playwright/src/test/java/com/microsoft/playwright/Utils.java [hunk 1]
```diff
@@ -122,4 +122,11 @@ static BrowserType getBrowserTypeFromEnv(Playwright playwright) {
         throw new IllegalArgumentException("Unknown browser: " + browserName);
     }
   }
+
+  static void verifyViewport(Page page, int width, int height) {
```
Comment (yury-s) [inline playwright/src/test/java/com/microsoft/playwright/Utils.java:126]: Can you update TestBrowserContextBasic to use this method and get rid of its local one?

File: playwright/src/test/java/com/microsoft/playwright/TestBrowserContextViewport.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    assertEquals(200, page.evaluate("() => window.screen.width"));
+    assertTrue((Boolean) page.evaluate("() => matchMedia('(min-device-width: 100px)').matches"));
+    assertFalse((Boolean) page.evaluate("() => matchMedia('(min-device-width: 300px)').matches"));
+    assertFalse((Boolean) page.evaluate("() => matchMedia('(max-device-width: 100px)').matches"));
+    assertTrue((Boolean) page.evaluate("() => matchMedia('(max-device-width: 300px)').matches"));
+    assertFalse((Boolean) page.evaluate("() => matchMedia('(device-width: 500px)').matches"));
+    assertTrue((Boolean) page.evaluate("() => matchMedia('(device-width: 200px)').matches"));
+    page.setViewportSize(500, 500);
+    assertEquals(500, page.evaluate("() => window.screen.width"));
+    assertTrue((Boolean) page.evaluate("() => matchMedia('(min-device-width: 400px)').matches"));
```
Comment (yury-s) [inline playwright/src/test/java/com/microsoft/playwright/TestBrowserContextViewport.java:54]: nit: you can avoid explicit cast with `assertEquals(true, page...` but it's up to you.

File: playwright/src/test/java/com/microsoft/playwright/TestBrowserContextViewport.java [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  @Test
+  void shouldSetTheProperViewportSize() {
+    verifyViewport(page, 1280, 720);
+    page.setViewportSize(123, 456);
+    verifyViewport(page,123, 456);
+  }
+
+  @Test
+  void shouldReturnCorrectOuterWidthAndOuterHeight() {
+    LinkedHashMap size = (LinkedHashMap) page.evaluate("() => {\n" +
```
Comment (yury-s) [inline playwright/src/test/java/com/microsoft/playwright/TestBrowserContextViewport.java:27]: Map<String, int> and avoid all the type casts below?

**Review by yury-s — APPROVED** [review]
Nice!

File: playwright/src/test/java/com/microsoft/playwright/Utils.java (hunk 1 above)
Comment (ephung01) [reply playwright/src/test/java/com/microsoft/playwright/Utils.java:126]: Will do

**Review by yury-s — APPROVED** [review]

**Pull request merged**